docs/appendices/release-notes/unreleased.rst

```diff
@@ -89,6 +89,10 @@ Changes
 Fixes
 =====
 
+- Fixed an issue in the PostgreSQL wire protocol that prevented values from
+  being serialized correctly if the client didn't send a ``describe`` message
+  and used the binary serialization format.
```

Comment:
does this really only affects the binary serialization?

Reply:
Mostly yes. Undefined maps to `VarChar` in the postgres layer and the text streaming there works for most types because of the implicit casts that are possible from `String -> <whatever>`